README.md

```diff
@@ -60,15 +60,15 @@ You will create a Ruby on Rails application that allows the user to:
 
 ## Live Demo <a name="live-demo"></a>
 
-Click to observe the [live project]()
+Click to observe the [live project](https://tariq-budget-app.onrender.com/)
 
 <p align="right"><a href="#readme-top">back to top</a></p>
 
 <!-- Live demo -->
 
 ## Presentation <a name="live-presentation"></a>
 
-Click to watch the [presentation on project]()
+Click to watch the [presentation on project](https://www.loom.com/share/e5f4e36b3f754b3389c6cb9e69024917)
```

Comment:
- [ ] Great work with the app, however, the video presentation doesn't have sound kindly check this out and make sure the video is working properly before using it for the presentation